dtls/examples/hub/src/utilities.rs

```diff
@@ -106,8 +106,9 @@ pub fn load_certificate(path: PathBuf) -> Result<Vec<CertificateDer<'static>>, E
     let f = File::open(path)?;
 
     let mut reader = BufReader::new(f);
-    match rustls_pemfile::certs(&mut reader) {
+    match rustls_pemfile::certs(&mut reader).collect::<Result<Vec<_>, _>>() {
```

Comment:
Use .collect::<Result<Vec<_>, _>>() to collect the iterator into a Result, which allows you to pattern match the Result